src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -137,11 +142,37 @@ class CPUSpecialSetup {
 };
 #endif // __linux__
 
+#if defined(OV_CPU_WITH_ACL)
+std::mutex Engine::SchedulerGuard::mutex;
+std::weak_ptr<Engine::SchedulerGuard> Engine::SchedulerGuard::ptr;
+
+Engine::SchedulerGuard::SchedulerGuard() {
+#if IE_THREAD == IE_THREAD_SEQ
+        arm_compute::Scheduler::set(arm_compute::Scheduler::Type::ST);
+#else
+        arm_compute::Scheduler::set(std::make_shared<ACLScheduler>());
+#endif
+}
+
+std::shared_ptr<Engine::SchedulerGuard> Engine::SchedulerGuard::instance() {
+    std::lock_guard<std::mutex> lock{SchedulerGuard::mutex};
+    auto scheduler_guard_ptr = SchedulerGuard::ptr.lock();
+    if (scheduler_guard_ptr == nullptr) {
+        SchedulerGuard::ptr = scheduler_guard_ptr = std::make_shared<SchedulerGuard>();
+    }
+    return scheduler_guard_ptr;
+}
+
+#endif
+
 Engine::Engine() :
     deviceFullName(getDeviceFullName()),
     specialSetup(new CPUSpecialSetup) {
     _pluginName = "CPU";
     extensionManager->AddExtension(std::make_shared<Extension>());
+#if defined(OV_CPU_WITH_ACL)
+    scheduler_guard = SchedulerGuard::instance();
```

Comment:
Do we really need `SchedulerGuard` now? Can we just write smt like that here:
```
std::lock_guard<std::mutex> lock{mutex};
arm_compute::Scheduler::set(std::make_shared<ACLScheduler>());
```

Reply:
@dmitry-gorokhov replaced

Reply:
@dmitry-gorokhov I reverted my new changes because this way:
```
std::lock_guard<std::mutex> lock{mutex};
arm_compute::Scheduler::set(std::make_shared<ACLScheduler>());
```
for RPI doesn't work 
I stayed `SchedulerGuard ` as worked solution for RPI